signers/signer-solana/package.json

```diff
@@ -21,6 +21,7 @@
     "lint": "eslint \"**/*.{ts,tsx}\" --ignore-path ../../.eslintignore"
   },
   "dependencies": {
+    "@rango-dev/wallets-shared": "^0.35.1",
```

Comment:
Please remove dependency to wallet shared for Solana signer. Signers should work independently.